sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableTestUtils.java

```diff
@@ -144,4 +146,24 @@ private static Cell createCell(ByteString value, long timestamp, String... label
     }
     return builder.build();
   }
+
+  public static BigtableIO.ReadChangeStream buildTestPipelineInput(
```

Comment:
if this is the only method that org.apache.beam.sdk.io.gcp.bigtable.changestreams.it refers to, it is not necessary to change the Util to public class. Just spin up the pipeline in place or Create BigtableChangeStreamTestUtils

Reply:
The problem, which we (bigtable team) dug ourselves into, is that `endTime` is a package private field. We don't want to expose `endTime` as an option because it's generally not well supported. But `endTime` is useful to terminate tests.

So we have to create the pipeline in the same package as `BigtableIO` so we actually can't create the pipeline with `endTime` in the `changestreams` package.

Reply:
ack. If there are other options needed then good to promote the scope of this Util class